vmdk_plugin/utils/refcount/refcnt.go

```diff
@@ -264,6 +264,19 @@ func (r *RefCountsMap) GetCount(vol string) uint {
 	return rc.count
 }
 
+//GetVolumeNames - return volume names of entries in refmap
```

Comment:
Yes. Just one additional optimization to get full volume name from entries in refmap(if there no multiple entries for given volume name) and avoid a trip to esx